nextcloudappstore/scaffolding/views.py

```diff
@@ -1,6 +1,7 @@
-from django.http import HttpResponse
-from django.views.generic import FormView
+from django.http import HttpResponse, Http404
```

Comment:
didn't you use `HttpNotFound` or so a few files above?

Reply:
I did add it it, but it wasn't used. I just need to remove the import there - will do.

Reply:
I was mostly confused by different naming for seemingly the same thing.